README.md

```diff
@@ -94,6 +94,27 @@ Flickr.authenticate(flickrOptions, function(error, flickr) {
 });
 ```
 
+### As flickrapi internally uses the request module, you can also pass default options for request that are accepted by request.defaults() as documented in the [request module](https://github.com/request/request#convenience-methods)
```

Comment:
let's point explicitly to https://github.com/request/request#requestdefaultsoptions here, since we push those options into `defaults()`, specifically.